pkg/controller/operators/catalog/operator.go

```diff
@@ -753,6 +753,7 @@ func (o *Operator) syncResolvingNamespace(obj interface{}) error {
 	}
 
 	// TODO: parallel
+	maxGeneration := 0
```

Comment:
Is this variable name a bit confusing? It implies max but it is more like current. Am I wrong here?

Reply:
This is intended to be the max generation among existing InstallPlans.

Reply:
I see. Thanks.